server/tracedb/connection/connection.go

```diff
@@ -38,11 +47,11 @@ func (r ConnectionTestStepResult) IsSet() bool {
 	return r.OperationDescription != ""
 }
 
-func IsReachable(endpoint string) (bool, error) {
-	if strings.HasPrefix(endpoint, "http://") || strings.HasPrefix(endpoint, "https://") {
+func IsReachable(endpoint string, protocol Protocol) error {
```

Comment:
Since we are not returning a `bool` anymore, it feels weird to call `IsReachable` to check an error. Makes sense to change this function to `CheckReachability` or something similar?